freqtrade/freqai/freqai_interface.py

```diff
@@ -105,6 +105,10 @@ def __init__(self, config: Config) -> None:
         self.max_system_threads = max(int(psutil.cpu_count() * 2 - 2), 1)
         self.can_short = True  # overridden in start() with strategy.can_short
         self.model: Any = None
+        if self.ft_params.get('principal_component_analysis', False) and self.continual_learning:
+            self.ft_params.update({'principal_component_analysis': False})
+            logger.warning('User tried to use PCA with continual learning. Deactivating PCA.')
```

Comment:
I'm fine with this change as is.

but this incompatibility seems/is new to me - so i think we should also mention it in the respective documentation sections.